Material.Blazor/npm.df.dockerignore

```diff
@@ -0,0 +1,10 @@
+**/*.csproj
```

Comment:
The dockerignore files are important to not get any "changed environment -> rebuild" effects using docker